src/openzaak/components/documenten/api/filters.py

```diff
@@ -30,13 +33,119 @@
 logger = logging.getLogger(__name__)
 
 
+class ZaakFilter(FkOrUrlFieldFilter):
+    def filter(self, qs, value):
+        if not value:
+            return qs
+
+        if not check_path(value, "zaken"):
+            logger.debug(
+                "URL (%s) is not a zaak, filtering to empty result set.", value
+            )
+            return qs.none()
+
+        return super().filter(qs, value)
+
+
 class EnkelvoudigInformatieObjectListFilter(FilterSet):
-    trefwoorden = CharArrayFilter(field_name="trefwoorden", lookup_expr="contains")
+    creatiedatum__gte = filters.DateFilter(
+        help_text=mark_experimental(
+            "De aanmakings datum van dit informatie object (groter of gelijk aan de gegeven datum)."
+        ),
+        field_name="creatiedatum",
+        lookup_expr="gte",
+    )
+    creatiedatum__lte = filters.DateFilter(
+        help_text=mark_experimental(
+            "De aanmakings datum van dit informatie object (kleiner of gelijk aan de gegeven datum)."
+        ),
+        field_name="creatiedatum",
+        lookup_expr="lte",
+    )
+    auteur = filters.CharFilter(
+        help_text=mark_experimental(
+            "De persoon of organisatie die dit informatie object heeft aangemaakt."
```

Comment:
Could you please add that this filter supports partial case insensitive match. We don't have it for other query params, so let's specify it explicitly.
The same comment applies for all `icontains` filters